### ISSUE TITLE
vagrant up fails `initialize': Permission denied

### ISSUE THREAD

**Issue by metaColin** [Varying-Vagrant-Vagrants/VVV#362]
I keep getting some sort of permission denied error, apparently related to Ruby:

```
==> default: Running cleanup tasks for 'shell' provisioner...
/Applications/Vagrant/embedded/gems/gems/vagrant-1.6.2/lib/vagrant/environment.rb:410:in `initialize': Permission denied - /Users/colin/.vagrant.d/data/lock.fpcollision.lock (Errno::EACCES)
```

It's completely torpedoed my efforts to get grunt running. I already tried

```
sudo chmod -R 755 /Applications/Vagrant
```

Made no difference...

Help!?

**Comment by f0ns** [comment]
I have the same issue, this did the trick for me:
sudo vagrant up

**Comment by brud** [comment]
vagrant 1.7.2 - same issue

**Comment by brud** [comment]
sudo chmod -R /home/<user_name>/.vagrant.d

it helps for me

**Comment by zamoose** [comment]
Don't do the sudo trick on your Mac or Linux build. Something's permissions
are probably off. Do a `find . -user root` from inside your VVV dir and see
if anything is off.

On Wed, Jan 14, 2015 at 2:41 PM, Anton Iliyn <notifications@github.com>
wrote:

> vagrant 1.7.2 - same issue
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/Varying-Vagrant-Vagrants/VVV/issues/362#issuecomment-69977895>
> .
>



-- 
-Doug

**Comment by svenhaveman** [comment]
Had the samen problem with the latest version of Vagrant:

opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:492:in `initialize': Permission denied - /Users/svenhaveman/.vagrant.d/data/lock.fpcollision.lock (Errno::EACCES)

I fixed this by changing the ownership of the file: chown svenhaveman:staff ~/.vagrant.d/data/lock.fpcollision.lock 

Don't sudo anything, this will lead to other errors.

**Comment by zachwills** [comment]
@svenhaveman fix worked for me- and I was running into it while getting VIP Quickstart set up

**Comment by lizkaraffa** [comment]
Same problem and I've tried various different installs hoping one was more stable.  I've tried @svenhaveman fix but no luck either :(

**Comment by keeprock** [comment]
@svenhaveman works for me too. I've got problems with id_rsa and private_keys files.

**Comment by cfoellmann** [comment]
I am not really informed when it comes to the progress of the vagrant development but there was one change that gave me some problems.

"Normally" all vagrants were using the ssh keys from the `$user/.vagrant` folder.
Now the keys are generated per vagrant and saved in the `.vagrant` folder connected to the specific vagrant

**Comment by alberto56** [comment]
This happened to me because I ran "vagrant up" with one user (user1 or root), then tried to run it with another (user2). When you attempt to vagrant up with user2, it can't access /Users/colin/.vagrant.d/data/lock.fpcollision.lock which is now owned by user1. It is maybe best to always run vagrant with the same user.

**Comment by lizkaraffa** [comment]
I always only use one user. I'm the only user on this computer.

**Comment by svenhaveman** [comment]
@lizkaraffa probably 99% of us have the same configuration as you. Can you check what the owner and permissions are from the /Users/lizkaraffa/.vagrant.d/data/lock.fpcollision.lock file?

**Comment by lizkaraffa** [comment]
@svenhaveman here's a screenshot of my permissions.  Please let me know if
there I'm missing something or didn't provide what you needed.

**Comment by svenhaveman** [comment]
I'm not seeing anything :). Can you upload the screenshot again?

**Comment by lizkaraffa** [comment]
@svenhaveman Sure here's a link to the screenshot: https://flic.kr/p/tYHfYf Let me know if I grabbed the right thing! :)

**Comment by svenhaveman** [comment]
This should be fine. You could try chmod'ing it to 755, but i would guess it doesn't make a difference. Maybe you have an other problem.

**Comment by lizkaraffa** [comment]
Yeah I just resorted back to a much older version and it works for me.
Anything newer and I have to be in sudo mode all the time which I don't
want to do

**Comment by danrcastro** [comment]
I had the same problem on OSX Yosemite today. @svenhaveman's tip saved the day.

**Comment by AlennGK** [comment]
Windows 10, same problem, but there is no fpcollision file!

**Comment by Committing** [comment]
Thanks @svenhaveman - worked for me

**Comment by AlexMazaltov** [comment]
In my case on Windows 10 i was looking in error that was in cmder. there was permission denied for C:/Users/Alex/.vagrant.d/tmp/hosts.local, i checked with ls -all there was two files:
`-r--r--r--  1 Alex 0 3466 2017-01-06 10:59 hosts.local`
`-rw-rw-rw-  1 Alex 0 2220 2017-01-06 11:31 hosts.youwe-vm` 
i removed falg "Read Only" for hosts.local it solved my issue

**Comment by noorcoder** [comment]
I am facing the same problem trying to install vagrant the last few days, no luck
Error:
C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/machine_index.rb:321:in `initialize': Permission denied @ rb_sysopen - C:/Users/Noor/.vagrant.d/data/machine-index/index.lock (Errno::EACCES)

My Configuration :
Local Host: Windows 10 Machine
Guest Host: ubuntu/trusty64

Your support is appreciate it!

**Comment by yourkin** [comment]
I had this issue by running sudo vagrant up on Mac and afterwards trying to run it without sudo (which, of course is the preferred way). Simply rm -rf .vagrant.d did the trick for me without any negative consequences.

**Comment by dallasclouddev** [comment]
> I had this issue by running sudo vagrant up on Mac and afterwards trying to run it without sudo (which, of course is the preferred way). Simply rm -rf .vagrant* did the trick for me without any negative consequences.

The solution posted by brianupload worked for me as well on Mac, Vagrant 2.6 and Virtual Box.

**Comment by tomjn** [comment]
TLDR: Never  use `sudo` with `vagrant`
